testing/tests/DevExpress.ui.widgets.scheduler/integration.appointments.tests.js

```diff
@@ -1321,9 +1280,9 @@ QUnit.test("Task dragging", function(assert) {
                 AllDay: false
             };
 
-            $(this.instance.$element()).find("." + APPOINTMENT_CLASS).eq(0).trigger(dragEvents.start);
+            let pointer = pointerMock($(this.instance.$element()).find("." + APPOINTMENT_CLASS).eq(0)).start().down().move(10, 10);
```

Comment:
Why you always need to move the mouse for 10px in both directions? If I drag appointment for 9px - it wouldn't work?